src/calculateRentalCost.js

```diff
@@ -7,4 +7,24 @@ function calculateRentalCost(days) {
   // write code here
 }
 
+const DAILY_RATE = 40;
+const LONG_TERM_RENTAL_DAYS = 7;
+const LONG_TERM_DISCOUNT = 50;
+const MID_TERM_RENTAL_DAYS = 3;
+const MID_TERM_DISCOUNT = 20;
+
+function calculateRentalCost(numberOfDays) {
```

Comment:
There is a duplicate declaration of the `calculateRentalCost` function. The first declaration (lines 1-8) should be removed or merged with this one.